src/modules/mavlink/mavlink_main.cpp

```diff
@@ -2229,6 +2229,14 @@ Mavlink::task_main(int argc, char *argv[])
 		send_autopilot_capabilities();
 	}
 
+#if defined(CONFIG_NET)
+
+	if (!multicast_enabled()) {
+		_src_addr_initialized = false;
+	}
+
+#endif // CONFIG_NET
+
```

Comment:
Is it really needed here? _src_addr_initialized is false by default and will be set to true upper if there is multicast option.

Reply:
I didn't think so, but I haven't dug into the original intent of the change. 

Let's try getting rid of it.